src/aks-preview/setup.py

```diff
@@ -23,7 +23,9 @@
     "License :: OSI Approved :: MIT License",
 ]
 
-DEPENDENCIES = []
+DEPENDENCIES = [
+    "holmesgpt @ git+ssh://git@github.com/mainred/holmesgpt@new-master", # will use a official pypi package once available
```

Comment:
The test branch from the forked repo will be replaced with official robusta-dev repo release.